Source/Controllers/MainViewControllers/ConnectionView/SPConnectionController.m

```diff
@@ -2244,17 +2243,15 @@ - (void)initiateSSHTunnelConnection
 
 	// Set up the tunnel details
 	sshTunnel = [[SPSSHTunnel alloc] initToHost:[self sshHost] port:[[self sshPort] integerValue] login:[self sshUser] tunnellingToPort:([[self port] length]?[[self port] integerValue]:3306) onHost:[self host]];
-	
-	if(sshTunnel == nil) {
-						[dbDocument setTitlebarStatus:NSLocalizedString(@"SSH Disconnected", @"SSH disconnected titlebar marker")];
```

Comment:
Did you remove the titlebar status? If so, can you add this back in some way?

Reply:
It was supposed to work only for SSH, but didn't as we removed that and the IBOutlet was disconenected.

I'm still unclear what was purpose of that, but once we gather more input it was necessary and what it included, we can figure something out.

Reply:
I'm still not sure WHY SSH should be showing another type of title anywhere else than other connections.

Reply:
FYI, I'm going to identify SSH connection - the accessory view (circle) will have `lock` icon in it if it's SSH. The accessory view is powerful thing and I think it will also look great :)

Reply:
Here ya go - https://github.com/Sequel-Ace/Sequel-Ace/pull/966